content/includes/footer.html

```diff
@@ -1,6 +1,5 @@
 <footer class="footer" role="contentinfo">
   <div class="footer-inner">
-    <a class="sr-only" href="#feedback-tool" name="feedback-tool">Give feedback on this page</a>
```

Comment:
@ncksllvn is this change consistent with your updates to the feedback tool and a reasonable way to resolve the link order issue that is referenced here?

Reply:
Yeah that looks good! That's consistent with the feedback tool. The only thing I had to watch out for was that the FAQ page contains this in its content -

` <a href="#feedback-tool" onclick="document.getElementById('feedback-tool').click()">Provide feedback on Vets.gov</a>.`

But this work isn't affected by that.

Reply:
Thanks for the heads up @ncksllvn.

Reply:
Thank you for your great work!